### ISSUE TITLE
Issue on Flatcar image

### ISSUE THREAD

**Issue by dmauser** [Microsoft-OpenHack/secure-networking-artifacts#2]
During the OH attendees get blocked by Azure Policy and also deployment fails over CLI because they must image accept the terms.
Here are the workarounds:

1) Disable the image policy enforcement
az policy assignment update --name "image-policy - Assignment" --enforcement-mode DoNotEnforce

2) Accept the terms for Kinvolk Flatcar image
az vm image terms accept --urn kinvolk:flatcar-container-linux-free:stable-gen2:latest

**Comment by erjosito** [comment]
Thanks for this issue @dmauser!

- I believe in the OH guides the bit about accepting the image is explicitly mentioned right?
- The point about the Azure Policy is not mentioned in the OH guide, but in the coach guides. Some coaches mentioned that it was a good exercise going to the policy and disable it, so we left it as it is. We could use an Ubuntu image instead (I recently updated the ARM templates with a `vmType` parameter that can take the values `kinvolk` or `ubuntu`).

If either of those remarks (the hint to accepting the terms in the OH guide or the one for the policy in the coaches guides) are not obvious enough, we should definitely improve that.

**Comment by dmauser** [comment]
Thanks for clarifying, @erjosito !
I got it. I missed that accept terms over the Coaches guide.
However, one of the feedbacks we got from the coaches and attendees in this round is to make sure deployments are smooth because troubleshooting it is outside of the scope of the networking, which is the goal of the OH. Similar scenario with CH6 for the App settings. Let's chat more about it. 
Closing this issue as resolved and let. Thanks again!
